UnitsNet.Tests/CustomCode/PressureChangeRateTests.cs

```diff
@@ -10,10 +10,22 @@ public class PressureChangeRateTests : PressureChangeRateTestsBase
 
         protected override double KilopascalsPerSecondInOnePascalPerSecond => 1e-3;
 
+        protected override double KilopoundsForcePerSquareInchPerMinuteInOnePascalPerSecond => 1.450377377302092e-7 * 60;
+
+        protected override double KilopoundsForcePerSquareInchPerSecondInOnePascalPerSecond => 1.450377377302092e-7;
+
         protected override double MegapascalsPerSecondInOnePascalPerSecond => 1E-6;
 
+        protected override double MegapoundsForcePerSquareInchPerMinuteInOnePascalPerSecond => 1.450377377302092e-10 * 60;
+
+        protected override double MegapoundsForcePerSquareInchPerSecondInOnePascalPerSecond => 1.450377377302092e-10;
+
         protected override double PascalsPerSecondInOnePascalPerSecond => 1;
 
+        protected override double PoundsForcePerSquareInchPerMinuteInOnePascalPerSecond => 1.450377377302092e-4 * 60;
+
+        protected override double PoundsForcePerSquareInchPerSecondInOnePascalPerSecond => 1.450377377302092e-4;
```

Comment:
Verified
https://www.google.com/search?q=Pascal+to+PoundsForcePerSquareInch+&sxsrf=ALeKk02wJ1E7cXU62Ee2WWoNFzaglNiC1Q%3A1617316553022&ei=yUpmYKFX6eKuBNe1pfgJ&oq=Pascal+to+PoundsForcePerSquareInch+&gs_lcp=Cgdnd3Mtd2l6EAM6BwgAEEcQsANQnrgBWJ_HAWC-yAFoAXABeACAAZ8BiAGZCpIBBDAuMTCYAQCgAQGgAQKqAQdnd3Mtd2l6yAEIwAEB&sclient=gws-wiz&ved=0ahUKEwjh34bVjd7vAhVpsYsKHddaCZ8Q4dUDCA0&uact=5